src/olympia/editors/models.py

```diff
@@ -306,9 +306,13 @@ def base_query(self):
 
     @property
     def authors(self):
+        def uniquify(sq):
+            see = set()
+            return [(x, y) for (x, y) in sq if x not in see and not see.add(x)]
```

Comment:
Are there cases when the id exists with different usernames? If there aren't you could just pass `sq` into `set` directly `return set(sq)`.

Otherwise mutating a set in a list comprehension feels wrong. I'd rather see this use a for loop directly.

```py
def uniquify(user_data):
    ids = set()
    users = []
    for id_, uname in user_data:
        if id_ not in ids:
            ids.add(id_)
            users.append((id_, uname))
    return users